$OpenBSD: patch-Makefile,v 1.7 2020/05/25 13:38:05 bket Exp $

Avoid use of pytest --with-doctest as this causes a regression test to fail
with "ImportError (Could not load inotify functions from libc)". Failure is
caused by the doctest module testing a piece of code that is linux-only. This
code is not used when running git-cola on OpenBSD.

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -71,7 +71,7 @@ endif
 FLAKE8_FLAGS = $(VERBOSE)
 
 PYTEST_FLAGS = $(QUIET) $(TEST_VERBOSE)
-PYTEST_FLAGS += --doctest-modules
+#PYTEST_FLAGS += --doctest-modules
 uname_S := $(shell uname -s)
 ifneq ($(uname_S),Linux)
     PYTEST_FLAGS += --ignore=cola/inotify.py
